web-ui/src/styles/_variables.css

```diff
@@ -0,0 +1,100 @@
+:root {
```

Comment:
What's the reason this filename starts with an underscore?

Reply:
I think this was just muscle memory from SCSS days, an underscore IIRC told the preprocessor to ignore outputting the file. If preferred then we could change this.